src/components/Liveness/index.js

```diff
@@ -39,22 +41,27 @@ export default class Liveness extends Component<CameraType, State> {
   }
 
   loadChallenges = () => {
-    this.setState({...initialState}, () =>
+    this.setState({...initialState, challengeRequestedAt: currentMilliseconds()}, () => {
       requestChallenges(this.props.token, this.handleResponse, this.handleError)
-    )
+      sendScreen(['face_video_challenge_requested'])
+    })
     this.setState({...initialState})
 
   }
 
   handleResponse = (response: Object) => {
     const {challenge, id} = response.data
     this.setState({ challenges: challenge, id, hasLoaded: true })
+    sendScreen(['face_video_challenge_loaded'], {challenge_loading_time: this.challenge_loading_time()})
   }
 
   handleError = () => {
     this.setState({ hasLoaded: true, hasError: true })
+    sendScreen(['face_video_challenge_load_failed'], {challenge_loading_time: this.challenge_loading_time()})
   }
 
+  challenge_loading_time = () => currentMilliseconds() - this.state.challengeRequestedAt
```

Comment:
Could we use camelCase instead of snake_case here?

Reply:
Yes :)